dracclient/client.py

```diff
@@ -753,6 +753,55 @@ def list_raid_controllers(self):
         """
         return self._raid_mgmt.list_raid_controllers()
 
+    def list_raid_cntrl_settings(self, by_name=False):
```

Comment:
Would you mind changing the above method name to:
list_raid_controller_settings()?

Reply:
done Chris.